__tests__/app.test.js

```diff
@@ -68,4 +68,16 @@ describe("my express project", () => {
 
 
 
+    describe("GET:/api/articles/id_not_number", () => {
```

Comment:
Ideally these error tests should all be in the same describe block for GET /api/articles/:article_id rather than in many separate describes